### ISSUE TITLE
Login not working

### ISSUE THREAD

**Issue by juanclb93** [bsobbe/iThenticate#1]
I tried to login using this package but it's not working.
I used same credentials on Ithenticate web and it works.

I tried this:
`$ithenticate = new Ithenticate("My username", "My password");`

But when I do `dd($ithenticate)`, result given is "false"

**Comment by bsobbe** [comment]
I think you need to contact Ithenticate to activate API access for your account.
Did you do that ?

**Comment by juanclb93** [comment]
No, I didn't. I did not think this were necessary. I can access through this web [https://api.ithenticate.com/en_us/login](url)

Thank you for your quick response.

**Comment by bsobbe** [comment]
Yes but as far as I know, API access is different from web access for accounts. Contact them and try to activate that.

**Comment by juanclb93** [comment]
I contacted Ithenticate and they say me all is fine now. But can't login yet. If it helps, I'm using laravel framework and I installed phpxmlrpc via composer.

My code is simply this:
`$ithenticate = new Ithenticate('username', 'pass');`
`dd($ithenticate);`

dd response is this:

`Ithenticate {#361 ▼`
`  -url: "https://api.ithenticate.com/rpc"`
`  -username: "username"`
`  -password: "pass"`
`  -sid: false`
`}`
